### PR TITLE
Update readme and add GitHub templates & codeowners

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: patch
Signed-off-by: Thodoris Greasidis <thodoris@balena.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,16 +108,24 @@ Run the test suite by doing:
 $ npm test
 ```
 
+In order to make the develop & test cycle faster, you can use [mocka's `.only` & `.skip`](https://mochajs.org/#exclusive-tests) variants
```
Comment (xginn8) [inline README.md]: Small typo here:
```suggestion
In order to make the develop & test cycle faster, you can use [mocha's `.only` & `.skip`](https://mochajs.org/#exclusive-tests) variants
```

**Comment by xginn8** [comment]
@thgreasi I stumbled upon this gem in the mocha docs today, might be worth considering adding as part of the precommit hook:

>     Be mindful not to commit usages of .only() to version control, unless you really mean it! To do so one can run mocha with the option --forbid-only in the continuous integration test command (or in a git precommit hook).

**Comment by thgreasi** [comment]
@xginn8 sounds like a great idea to add this in the `ci` script.
Do you want to open a PR for this??

**Comment by xginn8** [comment]
@thgreasi will do! opened https://github.com/balena-io/balena-sdk/issues/651 to track

**Pull request merged**